client/src/components/PitchDoc/ClaimPitchModal.tsx

```diff
@@ -0,0 +1,295 @@
+                  ))}
+                </Label.Group>
+              </div>
+              <div className="pitch-name"> {pitch.name} </div>
+              <span className="submitted-reviewed-by">
+                {' '}
+                Submitted By: {pitchAuthor}
+              </span>
+              <span className="submitted-reviewed-by">
+                {' '}
+                Reviewed By: {approvedBy}
+              </span>
+              <div className="section-description">
+                Here lies a two sentence summary of pitch. It will be two
+                sentences and no more.
+              </div>
+            </div>
+            <div className="pitch-link">
+              <div className="pitch-link-title">Link to Pitch:</div>
+              <Input
+                className="pitch-link-input"
+                placeholder="Pitch link"
+                onChange={(e) => {
+                  setPitchLink(e.currentTarget.value);
+                }}
+              />
+            </div>
+            <div className="role-contributor-section">
+              <div className="section-title">Roles Available Per Team:</div>
+              <div className="section-description">
+                Please select the team that you will be filling the role for
+                before claiming this pitch.
+              </div>
+              <div className="role-items">
+                {Object.entries(openTeams).map(([team, value], idx) => (
+                  <>
+                    <div className="role-item" key={idx}>
+                      <div className="checkbox">
+                        <Checkbox
+                          className="checkbox"
+                          onClick={() => handleCheckboxes(team)}
+                        />
+                      </div>
+                      <Label
+                        className="role-name"
+                        circular
+                        style={{
+                          backgroundColor:
+                            currentTeamsButtons[teamToTeamsButtons[team]],
+                        }}
+                      >
+                        {teamToTeamsButtons[team]}
+                      </Label>
+                      <div className="role-number">
+                        {value.target - value.current}
+                      </div>
+                    </div>
+                  </>
+                ))}
+              </div>
+              <div className="section-title">Pitch Claimed By:</div>
+              <div className="pitch-contributors">
+                {Object.entries(pitchContributors).map(
+                  ([name, pfpLink], idx) => (
+                    <>
+                      <Popup
+                        content={name}
+                        trigger={
+                          <img
+                            key={idx}
+                            src={pfpLink}
+                            className="pfp"
+                            alt="pfp"
```

Comment:
Super nit picky but our alt's should be a little more descriptive to cater to visually impaired users. A Text to Voice bot isn't going to know what to do with "pfp"